backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_Contract_Module_Log.java

```diff
@@ -1,246 +0,0 @@
-// Generated Model - DO NOT CHANGE
```

Comment:
Shouldn't you also remove the interfaces as well?

Reply:
yes. lemmi check, if was thinking i did that